src/components/Card.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/js/bootstrap.bundle.min.js';
+
+const Card = () => {
+  return (
+    <div>
+        <div className="card" style={{width: "20rem"}}>
```

Comment:
user col-3 or col-6 for the better responsive bootstrap code

Reply:
Its not working when i use col class.